elephant/test/test_causality.py

```diff
@@ -285,5 +286,223 @@ def test_non_zero_conditional_causality(self):
             self.non_zero_signal, 10, 'bic'), 0.0)
 
 
+class PairwiseSpectralGrangerTestCase(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    @staticmethod
+    def _generate_ground_truth_spectral(length_2d=300000, return_coeffs=False,
```

Comment:
I think this function should take the weights as a parameter, rather than hard-coded within the function.

Reply:
Addressed in [2d88ed4](https://github.com/NeuralEnsemble/elephant/pull/545/commits/2d88ed40ed32887170bab4616465be5bbdad5021).